### ISSUE TITLE
Crashes when removing images

### ISSUE THREAD

**Issue by hteumeuleu** [maizzle/framework#816]
- Maizzle Version: 4.2.1
- Node.js Version: 16.15.1

Hey there! One recurring issue I've had since using Maizzle is that whenever I touch to images within the `src/images` folder (from the Finder on macOS), the command line (`maizzle serve`) running in the background crashes. 

```
✖ ENOTEMPTY: directory not empty, rmdir 'build_local/images'
node:internal/process/promises:279
            triggerUncaughtException(err, true /* fromPromise */);
            ^

[Error: ENOTEMPTY: directory not empty, rmdir 'build_local/images'] {
  errno: -66,
  code: 'ENOTEMPTY',
  syscall: 'rmdir',
  path: 'build_local/images'
}
```

Not a big issue as I simply have to relaunch the command to run things again. But this could be a nice improvement given this is something I do very often!

**Comment by cossssmin** [comment]
Hey! This should be improved starting with `4.2.2` where we [fixed an issue](https://github.com/maizzle/framework/pull/783) with the content sources for Tailwind.

What was happening is that we were defining `./src/**/*.*` as a glob path for Tailwind to scan files for selectors to generate. Those paths commonly included `src/images` and it lead to two things: 1) Tailwind compilation was getting slower as the number of files in your images folder increased and 2) while Maizzle was building, Tailwind was accessing those files as well, sometimes causing a crash.

That `ENOTEMPTY` might happen for other reasons too, but do try with the latest version of Maizzle.

Note: looking at releasing `4.2.6` today which comes with [Tailwind CSS 3.2.0](https://tailwindcss.com/blog/tailwindcss-v3-2), so maybe hold off and update directly to that :)

**Comment by hteumeuleu** [comment]
Perfect, I'm closing this then!

**Comment by cossssmin** [comment]
Released `4.3.0`, check it out:

https://github.com/maizzle/framework/releases/tag/v4.3.0
